<launch>
    <arg name="image_topic" default="/usb_cam/image_raw" />
    <node pkg="amrl_motion_detection" type="motion_detection_node" name="motion_detection" output="screen" respawn="true">
        <remap from="image_topic" to="$(arg image_topic)" />
    </node>
</launch>